PROTOCOL.md

```diff
@@ -695,6 +708,15 @@ For example, given the schema string (pretty printed for readability. The entire
 
 Writers should reject any transaction that contains data where the expression `x > 3` returns `false` or `null`.
 
+## Check Constraints
+
+- Check constraints are stored at the table-level in `Metadata.configuration` and consist of a `name` and an `expression`. An input row satisfies a constraint when `expression` evaluates to `true`.
```

Comment:
do we need to say "stored at table-leve"? why not just say mentioned they are stored in the metadata configuration.